File: calculator.py
#!/usr/bin/env python3

# four functions

def multiply(a,b):
	return a*b

def add(a,b):
	return a + b

def subtract(a,b):
	return a - b

def divide(a,b):
	return a/b


#ask user input
a = eval(input("Enter the first number: "))
o = str(input("Enter the operation: "))
b = eval(input("Enter the second number: "))

#print explanations of the calculator

if o == "*":
	print("I'm going use the calculator functions to multiply " + str(a) + " and " + str(b))
	x = multiply(a,b)
	print(x)

if o == "+" :
	print("I'm going use the calculator functions to add " + str(a) + " and " + str(b))
	x = add(a,b)
	print(x)

if o == "-":
	print("I'm going use the calculator functions to subtract " + str(a) + " from " + str(b))
	x = subtract(a,b)
	print(x)

if o == "/":
	print("I'm going use the calculator functions to divide " + str(a) + " and " + str(b))
	x = divide(a,b)
	print(x)

File: calculator.js
function multiply(a,b) {
	return a*b
}

function add(a,b) {
	return a+b
}

function subtract(a,b) {
	return a-b
}

function divide(a,b) {
	return a/b
}

const prompt = require('prompt-sync')()

var a = prompt('Enter the first number: ')
var operation = prompt('Enter the operation (+, -, *, /): ');
var b = prompt('Enter the second number: ');

if (operation == "*") {
	console.log("I'm going use the calculator functions to multiply " + a +
		" and  " + b)
	var x = multiply(a,b)
	console.log(x)
}

if (operation == "+") {
	console.log("I'm going use the calculator functions to add " + a +
		" and  " + b)
	var x = add(a,b)
	console.log(x)
}

if (operation == "-") {
	console.log("I'm going use the calculator functions to subtract " + a +
		" from  " + b)
	var x = subtract(a,b)
	console.log(x)
}

if (operation == "/") {
	console.log("I'm going use the calculator functions to divide " + a +
		" by  " + b)
	var x = divide(a,b)
	console.log(x)
}